<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartChat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/img/new_logo.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="profileLabel">
        <property name="styleSheet">
         <string notr="true">color:rgb(50, 200, 200);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="friendsTab">
         <attribute name="title">
          <string>好友</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="friendsInfoLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QListWidget" name="friendsListWidget">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="chatButton">
              <property name="text">
               <string>聊天</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="usersTab">
         <attribute name="title">
          <string>用户</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <widget class="QLabel" name="usersInfoLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QListWidget" name="usersListWidget"/>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string>添加</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sendTab">
         <attribute name="title">
          <string>发送</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QListWidget" name="sendListWidget"/>
            </item>
            <item>
             <widget class="QLabel" name="sendInfoLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="sendTextEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="exitButton">
                <property name="text">
                 <string>退出</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sendFileButton">
                <property name="text">
                 <string>发送文件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sendButton">
                <property name="text">
                 <string>发送消息</string>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="recvTab">
         <attribute name="title">
          <string>接收</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QListWidget" name="recvListWidget"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="recvInfoLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="recvFileButton">
                <property name="text">
                 <string>接收文件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="recvButton">
                <property name="text">
                 <string>接收消息</string>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>群聊</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPlainTextEdit" name="sendAllTextEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="sendAllInfoLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="sendAllButton">
                <property name="text">
                 <string>发送消息</string>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="syncAction"/>
  </widget>
  <action name="syncAction">
   <property name="text">
    <string>同步</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
